cmd/mmlogicapi/apisrv/apisrv.go

```diff
@@ -438,8 +433,13 @@ func (s *mmlogicAPI) applyFilter(c context.Context, filter *mmlogic.Filter) (map
 	} else if count > 500000 {
 		// 500,000 results is an arbitrary number; OM doesn't encourage
```

Comment:
Do you think capping is better than not doing it at all?

Reply:
I did a couple different approaches to this during development, and I think from the perspective of OM being a tool used by matchmakers, that most of the time you'd prefer to know you had all the results or nothing? Otherwise the MMF might make a match that it would otherwise reject as not high enough quality if it had the whole matching player pool?  Ultimately I think that we just need to make it clear that folks should be constructing pools that they can fully retrieve, which ultimate is a subjective evaluation about how long you're willing to wait?  Like, you can actually retrieve a pool of this size just fine, it just takes several seconds.  If your matchmaking is okay with waiting that long, you could just adjust this number higher or even comment this branch out entirely?  We just don't want a situation where OM lets you 'accidentally' shoot yourself in the foot WRT performance or incomplete filter results.  Also compounded as either of those situations would be easy for inexperienced users to misinterpret as 'Open Match is broken'.  Thoughts?